docs/semgrep-language-reference/semgrep-rule-schema.yaml

```
@@ -0,0 +1,836 @@
+        then:
+          required: [id, message, languages, severity, pattern-sources, pattern-sinks]
+
+      # Search mode (explicit or implicit default).
+      - if:
+          properties:
+            taint: false
+            join: false
+            extract: false
+          allOf:
+            - anyOf:
+                - properties: { mode: { const: search } }
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)